Tests/MiniatureTests/MiniatureTests.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  MiniatureTests.swift
+//  MiniatureTests
+//
+//  Created by Alex on 6/19/25.
+//  Copyright © 2025 WordPress. All rights reserved.
+//
+
+import Testing
+@testable import Miniature
+
+struct MiniatureTests {
+
+    @Test func example() async throws {
+        // Write your test here and use APIs like `#expect(...)` to check expected conditions.
+    }
+
+}
```

Comment:
I guess we don't need test targets for the Miniature app?

Reply:
I think we might in case you need to add a new testable component to the app targets and want to quickly iterate on it before moving it. It doesn't hurt to have this test target around.